### DISCUSSION TITLE
What firmware is safe to flash with Carlinkit cpc200-cp2a?

### DISCUSSION THREAD

**Discussion by vanoir** [ludwig-v/wireless-carplay-dongle-reverse-engineering#143] [General]
Hello, 

I recently purchased this dongle, it worked perfectly half of the time, sometimes I had to replug the car USD.

Then I ran the regular update and now it only shows a constant red light. Nothing on car screen and no wifi/bluetooth networks.

Seems flashing a new firmware is the way to go but Im not sure which one to pick.

**Comment by team-learncity** [comment]
Sorry, you can delete this, I see now this repo is for a different device kind
